pkg/controller/controller.go

```diff
@@ -483,6 +486,22 @@ func StartController(ctx context.Context, cfg *config.Config, defaultNamespace s
 		return errors.Wrapf(err, "error building FlyteWorkflow clientset")
 	}
 
+	// Create FlyteWorkflow CRD
+	apiextensionsClient, err := apiextensionsclientset.NewForConfig(kubecfg)
+	if err != nil {
+		return errors.Wrapf(err, "error building apiextensions clientset")
+	}
+
+	logger.Infof(ctx, "Creating FlyteWorkflow CRD")
+	_, err = apiextensionsClient.ApiextensionsV1().CustomResourceDefinitions().Create(ctx, &flyteworkflow.CRD, v1.CreateOptions{})
```

Comment:
One problem here is that, we will need permissions to create a CRD. Can we instead do the following
```python
if get_crd("name") is None:
     err = create("name")
     if err is AlreadyExists:
          return
```

Reply:
this way, we always have a get, but do not need a create permissions. Someone can create the CRD externally

Reply:
Done.